pootle/templates/languages/admin/language_team.html

```diff
@@ -12,7 +12,8 @@
     <div id="pootle-fs-project-admin" class="settings-container">
       <div class="config-content">
 	<div class="config-content-panel">
-	  <h3>{% blocktrans %}Member admin{% endblocktrans %}</h3>
+	  {% comment %}Translators: This means management for team members{% endcomment %}
```

Comment:
A comment is always "This mean" so avoid

Administration or management of the members of a team